### PR TITLE
[Rekognition] Add DetectModerationLabels API

### PR DESCRIPTION
**Pull request by t-richard**
This adds the DetectModerationLabels that allows to detect inappropriate images

See: 
- https://docs.aws.amazon.com/rekognition/latest/APIReference/API_DetectModerationLabels.html
- https://docs.aws.amazon.com/rekognition/latest/dg/procedure-moderate-images.html

Will try this for real later this weekend

### CODE REVIEW

**Comment by t-richard** [comment]
Just tested it live and it works as expected for my use case !

File: manifest.json [hunk 1]
```diff
@@ -430,6 +430,7 @@
                 "DeleteCollection",
                 "DeleteProject",
                 "DetectFaces",
+                "DetectModerationLabels",
```
Comment (jderusse) [inline manifest.json:433]: Could you add an entry in the `src/Service/Rekognition/CHANEGELOG.md` file :pray:

**Review by jderusse — APPROVED** [review]

File: manifest.json (hunk 1 above)
Comment (t-richard) [reply manifest.json:433]: Done, there was already a `NOT RELEASED` entry in the changelog so I added it there, let me know if you want me to write a specific version (probably 0.1.8)

**Review by jderusse — APPROVED** [review]

File: manifest.json (hunk 1 above)
Comment (jderusse) [reply manifest.json:433]: perfect

**Comment by jderusse** [comment]
thank you @t-richard

**Comment by t-richard** [comment]
Thanks for the great work on async-aws :heart: 

Love how easy it was to add a new operation and contribute it !

**Pull request merged**